server/routing/router_test_help.go

```diff
@@ -24,19 +25,24 @@ func startTestServer() (int, *services.MasterList) {
 	return port, svcs
 }
 
-func get(port int, endpoint string, data []byte) ([]byte, error) {
-	return doHTTPReq(http.MethodGet, port, endpoint, data)
+func get(port int, endpoint string, msg interface{}) ([]byte, error) {
+	return doHTTPReq(http.MethodGet, port, endpoint, msg)
```

Comment:
This now means we can just pass around anything, and it will be Marshaled automatically. We still have to unmarshal the response properly, but at least one side of it is magic now.